tests/functional_small/ops/ops_tpetra_vector.cc

```diff
@@ -386,62 +333,112 @@ TEST_F(tpetraVectorGlobSize15Fixture, vector_update4_b)
     }
 }
 
-TEST(ops_tpetra, vector_elementwiseMultiply)
+TEST_F(ops_tpetra, vector_update_nan1)
+{
+    auto v = pressio::ops::clone(*myVector_);
+    auto v2_h = v.getLocalViewHost(Tpetra::Access::ReadOnly);
+    auto v_h = Kokkos::subview(v2_h, Kokkos::ALL, 0);
+    pressio::ops::fill(v, 1.);
+    auto a = pressio::ops::clone(*myVector_);
+    pressio::ops::fill(a, 1.);
+    auto nan = pressio::ops::clone(*myVector_);
```

Comment:
can you please rename: `nan -> vecOfNan`

Reply:
ok, fixed on 5bcb6c5b0b9e618cc4bf51aa0cdcd7b0b314b274